domains/nullifiedsec.json

```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "NullifiedSec",
+           "email": "",
+           "discord": "1071358672342626324",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.hbufBywD3-tiOABuE3DrN99Cz4LYAxm9F35wlhmjn4Y7D70fCioNgMiF4f8eZU0DilSW28mj1hehqN6cSwjziz0R-5jU5PtFh-P_6qOvexlwoj1WXcdqVQDjMYODkDRejwp8HX4O319ypfgdNGyWYm0fe6mVOfzUM-sGD0QkRJABGmsfXQrG_fFaQNTqDm6KfBJlq6rTmr8ebqIjhfRYmhORK_wWGBrKgISScUB64OqtgzDJaaijeHbWEIdMXrkOYrXoSdNTMCT_5Js2pwHW0V3Az2MefdKtCRG2u-ZxBwkSk7iAj5MCAwPV1zYz3G3fNjpmFFrec-bkQggORgR4_w.zeiQDxqyKZy79RUl6LKWiA.e9oE3qmaZdhxCET31HAqvSPLyPHHfhX6Jr7-B3dkp_TZddMrfy0ZXvGSOioFrpcbAPSp7NP06CAYMhg3rVP7Bre0Y-AHFr8UyG77MFypDmU.xYG6reTv77v6L5CPSCIV1Q"
+        },
+    
+        "record": {
+            "A": ["185.199.108.153"]
```

Comment:
Github Pages needs 4 A records

Reply:
Sorry Actually I am Not Really Into That Domain Stuff Can You Please Help Me ?